source/manual/howto-checkout-and-commit-to-codecommit.html.md

```diff
@@ -0,0 +1,90 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: How to check and commit code to AWS codecommit
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+---
+Codecommit is used as a backup/mirror to our repositories in github as such we don't need to push code there all the time.
+We can leverage the access granted to us via the GDS cli in order to authenticate our local user and commit code.
```

Comment:
```suggestion

Should we need to commit code manually to CodeCommit (if GitHub is down, for example), we can leverage the access granted to us via [gds-cli](https://github.com/alphagov/gds-cli) in order to authenticate our local user and commit code.
```

Though given we can't deploy directly from CodeCommit, I am struggling to think of a situation where we'd actually need to commit code to CodeCommit... perhaps you can think of a better example above? 🤔

Reply:
I still wonder if a better example exists, @martynipratt? In other words, what's the main driver for us having the ability to push code to CodeCommit? It had its uses when we had a Jenkins job that communicated with it and allowed us to deploy from CodeCommit if GitHub was down, but now we've moved to Argo (which also relies on GitHub) our fallback deployment mechanism is different. I can see the benefit of CodeCommit as a mirror, but still not sure of a use case for pushing to it when GH is down.

Reply:
Now that we have Github for both source control and pipeline it seems wise to consider having a contingency for the pipeline part as well. 

I see the scope of the PR/ticket be to replace the previous process of using shared credentials. 
I've created another [ticket](https://trello.com/c/5eMaxzB5/2986-move-the-current-codecommit-backup-off-of-jenkins) to work on updating the codecommit backup process which depends on puppet and Jenkins anyway so will need to be updated. 
I and could add another ticket to look into adding contingency for github actions in the event that Github actions goes down. I see that being a significantly large piece of work that is outside the scope of this PR/ticket.